### ISSUE TITLE
Dev: fix Error: Broken ARIA reference on Security page

### ISSUE THREAD

**Issue by ExperimentsInHonesty** [hackforla/tdm-calculator#3045]
### Overview
We need to fix the Error(s): Broken ARIA reference on Security so that page can be WCAG compliant.

#### Details
Category: Error
Name: Broken ARIA reference
(The Accessibility wiki for this Error has a solution)

Page that will be worked on:
Security

### Action Items
- [x] Review wiki page: Accessibility, How to run the WAVE browser extension (Resource 1.01)
   - [x] Install the WAVE browser extension
- [x] Log in as the user indicated (resource 1.02.01) 
- [x] navigate to the page you are going to fix (resource 1.02.02)
- [x] Open the TDM WAVE Accessibility Audit Spreadsheet, for this Error (resource 1.03) and find the row that relates to this  issues page.
- [x] Run the WAVE extension with no elements focused, or clicked
- [x] Find the Error referenced in the Spreadsheet (1.03)
- [x] This Error has been resolved on other pages, so review `Project Team Solution` section (resource 1.04.01.01)
- [x] Apply solution
- [x] Submit PR
- [x] Add PR to spreadsheet in PR column next to Error (resource 1.03)
- [x] On this issue
   - [x] Add labels: 
      - `ready for accessibility lead`
   - [x] Change status to: `Technical Review`
   - to the epic #2891
     - [x] Add the label `ready for product`
      - [x] Add this comment 
         ```
          - #3045 is closed.  Please review to see if there are any remaining instances of this problem
         ```

### Resources/Instructions
- 1.01 [How to run the WAVE browser extension](https://github.com/hackforla/accessibility/blob/main/docs/getting-started/wave-browser-extension.md)
- 1.02 Application details
   - 1.02.01 User Profile Required: security admin
   - 1.02.02 Page URL: https://tdm-dev.azurewebsites.net/roles
      - Requires sign in: TRUE
      - Can access page directly from URL: FALSE
- 1.03 [TDM WAVE Accessibility Audit Spreadsheet, tab: 2. Audit Log, filter: Error: Broken ARIA reference](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0&fvid=713278591)
   -  Spreadsheet Row(s): 11
- 1.04 [Accessibility Known Issues Index](https://github.com/hackforla/accessibility/blob/main/docs/known-issues-and-solutions/solutions-index/index.md)
    - 1.04.01 [WAVE Known Accessibility Issue: Error - Broken ARIA reference](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/error-broken-aria-reference/)
        - 1.04.01.01 [WAVE Known Accessibility Issue: Error - Broken ARIA reference, Project Team Solution](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/error-broken-aria-reference/#project-team-solution)

Additional reading (optional)
- https://hackforla.github.io/accessibility/

#### Slides
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
   - [Staging Deck, slide 3045](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg/edit?slide=id.g3d52b301129_41_0#slide=id.g3d52b301129_41_0)
- Release deck
   - [2026-MM-DD Release deck, slide 3045]

**Comment by Rabia2219** [comment]
@geolunalg 
Please add a level label. You can choose from one of the following:

- `good first issue`
- `level: easy`
- `level: medium`
- `level: hard`

Once you have added the level label, please:
- remove the `level: missing` label
- remove the `ready for accessibility lead` label
- add the `ready for prioritization` label

**Comment by Philc90** [comment]
Verified no WAVE Errors appear on the Security page, appears to have been fixed already.

<img width="1856" height="916" alt="Image" src="https://github.com/user-attachments/assets/dfcf52c8-db41-488c-b9a3-99d36409d730" />

**Comment by geolunalg** [comment]
This was address in the first round of accebility audit, not PR needed.

`- #3045 is closed.  Please review to see if there are any remaining instances of this problem`

**Comment by ExperimentsInHonesty** [comment]
@geolunalg @Philc90 
- I updated the spreadsheet, and am closing this issue. The fix was made on issue #2709.  Contrary to instructions below do not expand scope of this issue
- 
>- #3045 is closed.  Please review to see if there are any remaining instances of this problem
